src/components/confirmation-dialogs.yml

```diff
@@ -20,10 +20,10 @@ deleteRoomConfirmation:
   en: Do you really want to delete room "{roomName}"?
   de: Möchtest du den Raum "{roomName}" wirklich löschen?
 deleteRoomMemberConfirmation:
-  en: Do you really want remove member "{memberDisplayName}" from the room?
+  en: Do you really want to remove member "{memberDisplayName}" from the room?
   de: Möchtest du das Mitglied "{memberDisplayName}" wirklich vom Raum entfernen?
 deleteRoomInvitationConfirmation:
-  en: Do you really want revoke room invitation send to "{invitationEmail}"?
+  en: Do you really want to revoke room invitation send to "{invitationEmail}"?
```

Comment:
Actually, not related to the ticket, but there's another type, it should be:

> Do you really want to revoke room invitation sen**t** to "{invitationEmail}"?

Reply:
Definitely, I had missed that one. Changed!